app/Http/Controllers/EpisodeManager.php

```diff
@@ -122,4 +155,92 @@ public function uploadFileToS3(Request $request)
 
         return $s3->getDriver()->getAdapter()->getClient()->getObjectUrl('suyabay', $fileName);
     }
+
+    public function delete( Request $request )
```

Comment:
Remove the spaces around the paramaters. It shouldnt be there.

It should be `public function delete(Request $request)`